book/src/advanced_database.md

```diff
@@ -72,17 +83,19 @@ The historical state cache size can be specified with the flag `--historic-state
 lighthouse beacon_node --historic-state-cache-size 4
 ```
 
-> Note: This feature will cause high memory usage.
+> Note: Use a large cache limit can lead to high memory usage.
 
 ## Glossary
 
-* _Freezer DB_: part of the database storing finalized states. States are stored in a sparser
+- _Freezer DB_: part of the database storing finalized states. States are stored in a sparser
   format, and usually less frequently than in the hot DB.
-* _Cold DB_: see _Freezer DB_.
-* _Hot DB_: part of the database storing recent states, all blocks, and other runtime data. Full
+- _Cold DB_: see _Freezer DB_.
+- _HDiff_: hierarchical state diff.
+- _Hierarchy Exponents_: configuration for hierarchical state diffs, which determines the density
+  of stored diffs and snapshots in the freezer DB.
+- _Hot DB_: part of the database storing recent states, all blocks, and other runtime data. Full
   states are stored every epoch.
-* _Restore Point_: a full `BeaconState` stored periodically in the freezer DB.
-* _Slots Per Restore Point (SPRP)_: the number of slots between restore points in the freezer DB.
-* _Split Slot_: the slot at which states are divided between the hot and the cold DBs. All states
+- _Snapshot_: a full `BeaconState` stored periodically in the freezer DB. Yearly by default.
```

Comment:
should be every 291 days?

Reply:
Good catch! https://github.com/sigp/lighthouse/pull/5978/commits/c4d6ef3544b529bbbc8a919916165e6e976eb1fb